src/Context/Node/NodeContext.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @author Sergii Bondarenko, <sb@firstvector.org>
+ */
+namespace Drupal\TqExtension\Context\Node;
+
+use Drupal\TqExtension\Context\RawTqContext;
+
+class NodeContext extends RawTqContext
+{
+    /**
+     * @When /^I (visit|view|edit) "([^"]+)" node of type "([^"]+)"$/
+     */
+    public function iVisitNodePageOfType($operation, $title, $type) {
+        if ($operation == 'visit') {
```

Comment:
In this package, as in Symfony, we're using left-hand operator comparison. This prevent from wrong assignment. Also, three equal sign should be used to tell the interpreter to not check the types of operands.

Example:

```php
if ('visit' === $operation)
```